packages/storybook/.storybook/manager.js

```diff
@@ -1,20 +1,40 @@
 import { addons } from '@storybook/addons';
 import React from 'react';
+import webComponentDocs from '@department-of-veterans-affairs/web-components/component-docs.json';
 import { additionalDocs } from '../stories/additional-docs';
 import { category } from '../stories/maturity-scale';
 
+const getMaturityCategory = (componentName) => {
+  // find the docs object for the component
+  const componentDocs = webComponentDocs.components.filter(item => (
+    item.docsTags.some(tag => tag.text === componentName)
+  ));
+  
+  // get the maturity category for the component
+  const maturityCategory = componentDocs[0]?.docsTags
+    .filter(item => item.name === 'maturityCategory')
+    .map(item => item.text)
+    .toString()
+    .toUpperCase();
+  
+  return category[maturityCategory];
+}
```

Comment:
Would it make sense to reuse helpers for this? There's that `getDocsTagValue` that you wrote as well as this:

https://github.com/department-of-veterans-affairs/component-library/blob/ccc6d2d88c567db08e6e17d512230dea8a76f7f4/packages/storybook/stories/wc-helpers.jsx#L16-L20

That function could probably be refactored to use the `componentName` like we have here.